client/src/components/game/phases/Events.vue

```diff
@@ -18,7 +18,10 @@
           </div>
         </div>
       </div>
-      <div class="active-events col-8 tour-active-events">
+      <div class="active-events tour-active-events"
+        v-bind="{class: currentEvent.clientViewHandler == 'INFLUENCES_SELECT' ||
```

Comment:
If we push the conditional logic into a function `shouldShowAccomplishments`, we can reuse it to make the picking the CSS class selection a little simpler. Also you'll only need to update it in one place if it ever changes.